ass_man/models.py

```diff
@@ -20,62 +21,84 @@ def __str__(self):
+    u16 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset16')
+    u17 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset17')
+    u18 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset18')
+    u19 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset19')
+    u20 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset20')
+    u21 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset21')
+    u22 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset22')
+    u23 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset23')
+    u24 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset24')
+    u25 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset25')
+    u26 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset26')
+    u27 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset27')
+    u28 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset28')
+    u29 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset29')
+    u30 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset30')
+    u31 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset31')
+    u32 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset32')
+    u33 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset33')
+    u34 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset34')
+    u35 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset35')
+    u36 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset36')
+    u37 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset37')
+    u38 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset38')
+    u39 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset39')
+    u40 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset40')
+    u41 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset41')
+    u42 = models.ForeignKey(Asset, on_delete=models.SET_NULL, blank=True, null=True, related_name='Asset42')
```

Comment:
As an example for delete behavior, this is saying 

when the foreignkey Asset is deleted, what should happen to the field u42? and the answer is that it should be set to null